lib/teslamate/vehicles/vehicle/summary.ex

```diff
@@ -11,7 +11,7 @@ defmodule TeslaMate.Vehicles.Vehicle.Summary do
     speed outside_temp inside_temp is_climate_on is_preconditioning locked sentry_mode
     plugged_in scheduled_charging_start_time charge_limit_soc charger_power windows_open doors_open
     odometer shift_state charge_port_door_open time_to_full_charge charger_phases
-    charger_actual_current charger_voltage version update_available is_user_present geofence
+    charger_actual_current charger_voltage version update_available update_version is_user_present geofence
```

Comment:
Fields added here are automatically published via MQTT. Could you add the new `update_version` topic to the MQTT docs (https://github.com/adriankumpf/teslamate/blob/master/website/docs/integrations/mqtt.md)?